src/style.css

```
@@ -1 +1,44 @@
 /* add styles here */
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.stars--1 :nth-child(-n + 1) {
```

```suggestion
.stars--1 :nth-child(-n + 1),
.stars--2 :nth-child(-n + 2),
... {
```
Consider using commas to group your selectors